perun-core/src/main/java/cz/metacentrum/perun/core/api/MembersManager.java

```diff
@@ -1271,6 +1271,17 @@
 	 */
 	List<RichMember> getSponsoredMembers(PerunSession sess, Vo vo) throws PrivilegeException, VoNotExistsException;
 
+	/**
+	 * Gets list of sponsored members with sponsors.
+	 *
+	 * @param sess session
+	 * @param vo virtual organization from which are the sponsored members chosen
```

Comment:
Please add ``attrNames`` param to the javadoc.

Reply:
I added this param to javadoc.